resources/gemini-1b/utils/scripts/generate_node_keys.sh

```diff
@@ -0,0 +1 @@
+../../../../scripts/generate_node_keys.sh
```

Comment:
Why not just move files here? I don't think they're fully reusable anyway, so copying makes sense.

Reply:
its used for x-net

Reply:
Okay, but the question still remains unanswered...